### PR TITLE
Fixing routing app so it no longer uses deprecated methods.

### PR DESCRIPTION
**Pull request by mbcoder**
Also changed the list so it displays the route instructions instead of
the street name.

@tyle8552 can to review this

### CODE REVIEW

File: src/main/java/com/esri/samples/na/find_route/FindRouteSample.java [hunk 1]
```diff
@@ -175,9 +176,10 @@ public void start(Stage stage) throws Exception {
             Point stop2Loc = new Point(-1.3036911787723785E7, 3839935.706521739, ESPG_3857);
 
             // add route stops
-            List<Stop> routeStops = routeParameters.getStops();
+            List<Stop> routeStops = new ArrayList<Stop>();
```
Comment (ghost) [inline src/main/java/com/esri/samples/na/find_route/FindRouteSample.java]: You can omit the Stop type in ArrayList.

**Review by ghost — APPROVED** [review]
Looks good. Just one suggestion.

**Comment by mbcoder** [comment]
Thanks @tyle8552 I've updated that. Okay to merge now?

File: src/main/java/com/esri/samples/na/find_route/FindRouteSample.java [hunk 2]
```diff
@@ -109,10 +110,10 @@ public void start(Stage stage) throws Exception {
           routeGraphic = new Graphic(shape, new SimpleLineSymbol(SimpleLineSymbol.Style.SOLID, BLUE_COLOR, 2));
           routeGraphicsOverlay.getGraphics().add(routeGraphic);
 
-          // get route street names
-          route.getDirectionManeuvers().stream().flatMap(mvr -> mvr.getManeuverMessages().stream()).filter(ms -> ms
-              .getType().equals(DirectionMessageType.STREET_NAME)).forEach(st -> directionsList.getItems().add(st
-                  .getText()));
+          // get the direction text for each maneuver
+          for(DirectionManeuver step : route.getDirectionManeuvers()) {
```
Comment (colinanderson) [inline src/main/java/com/esri/samples/na/find_route/FindRouteSample.java]: Should be `for (DirectionManeuver step : route.getDirectionManeuvers())` with a space between for and the bracket. I'm sure you could replace the loop with some stream based thing though.

**Comment by mbcoder** [comment]
thanks @colinanderson I've fixed up formatting.

**Comment by ghost** [comment]
@mbcoder Looks good to merge.

**Pull request merged**